recipes/opusfile/all/conanfile.py

```diff
@@ -55,10 +56,10 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("ogg/1.3.5")
-        self.requires("opus/1.3.1")
+        self.requires("ogg/1.3.5", transitive_headers=True)
```

Comment:
IT would have been nice to comment https://github.com/conan-io/conan-center-index/pull/17080#issuecomment-1512849220 to show it was required :)